konfera/settings.py

```diff
@@ -44,3 +44,9 @@
 Universal BCC for all notifications
 """
 EMAIL_NOTIFY_BCC = getattr(settings, 'EMAIL_NOTIFY_BCC', [])
+
+
+"""
+Number of days after the user is notified about unpaid order
+"""
+UNPAID_ORDER_NOTIFICATION_DAYS = getattr(settings, 'UNPAID_ORDER_NOTIFICATION_DAYS', 3)
```

Comment:
Lets change the default to 5 days.

I am thinking if we can make it, a little bit more flexible with minor effort. How about we add two more options:

* `UNPAID_ORDER_NOTIFICATION_REPEAT = 3  # How many times we should repeat the email notification`
* `UNPAID_ORDER_NOTIFICATION_REPEAT_DELAY = 5  # delay in days`